images/cli/Dockerfile.rhel

```diff
@@ -8,5 +8,5 @@ COPY --from=builder /go/src/github.com/openshift/oc/oc /usr/bin/
 RUN for i in kubectl openshift-deploy openshift-docker-build openshift-sti-build openshift-git-clone openshift-manage-dockerfile openshift-extract-image-content openshift-recycle; do ln -sf /usr/bin/oc /usr/bin/$i; done
 LABEL io.k8s.display-name="OpenShift Client" \
       io.k8s.description="OpenShift is a platform for developing, building, and deploying containerized applications." \
-      io.openshift.build.versions="kubectl=1.30.1" \
+      io.openshift.build.versions="kubectl=1.30.2" \
```

Comment:
We need to update tools image as well https://github.com/openshift/oc/blob/5e45e289cc56aa7a29533c8438e1fe72ee6de87c/images/tools/Dockerfile#L57

Reply:
Updated. Thx.